__tests__/bookshop/ord/custom.ord.json

```diff
@@ -0,0 +1,51 @@
+{
+    "namespace": "should not update since has conflict with cdsrc.json",
+    "openResourceDiscovery": "should not update since not defined in cdsrc.json",
+    "groups": [
+        {
+            "groupId": "sap.cds:service:sap.test.cdsrc.sample:AdminService",
+            "groupTypeId": "sap.cds:service",
+            "title": "new Admin Service"
+        }
+    ],
+    "apiResources": [
+        {
+            "apiProtocol": "odata-v5",
+            "description": "new AdminService",
+            "ordId": "sap.test.cdsrc.sample:apiResource:AdminService:v1",
+            "entityTypeMappings": [
+                {
+                    "entityTypeTargets": [
+                        "new.entityType.target"
+                    ]
+                }
+            ]
+        }
```

Comment:
Better have a testcase, where you add a new rest API, the other API resource overrides should be handled via @ORD.Extension mechanism, see linked issue.